assistant/assistant.go

```diff
@@ -251,27 +299,55 @@ func (assistant *Assistant) updatePlugin(req request) response {
 	return res
 }
 
-// TODO Implement after merge
-func (assistant *Assistant) getRunningPlugins() (response, error) {
-	// inputs := assistant.agent.GetInputPlugins()
-	// outputs := assistant.agent.GetOutputPlugins()
-	// data := map[string][]string{
-	// 	"outputs": outputs,
-	// 	"inputs":  inputs,
-	// }
-	// res = response{SUCCESS, req.UUID, data}
-	// return res
-	return response{}, nil
+// stopPlugin stops a single plugin
+func (assistant *Assistant) stopPlugin(req request) response {
+	fmt.Print("D! [assistant] Received request: ", req.Operation, " for plugin ", req.Plugin.Name, "\n")
+
+	var res response
+	var err error
+
+	switch req.Plugin.Type {
+	case "INPUT":
+		assistant.agent.StopInputPlugin(req.Plugin.Name)
```

Comment:
Shouldn't you get some kind of error back so we know if it occurred successfully or not?